pephub/routers/api/v1/project.py

```diff
@@ -151,20 +139,17 @@ async def update_a_pep(
                 status_code=400,
                 detail=f"Could not create PEP from provided yaml. Error: {e}",
             )
-        # validate each sample in the table according to the project
-        for s in new_project.samples:
-            sample_name: str = s[new_project.st_index]
-            try:
-                eido.validate_sample(
-                    new_project,
-                    sample_name,
-                    "http://schema.databio.org/pep/2.0.0.yaml",  # just use the base PEP schema for now
-                )
-            except Exception as e:
-                raise HTTPException(
-                    status_code=400,
-                    detail=f"Sample {sample_name} failed validation: {e}",
-                )
+
+        try:
+            # validate project (it will also validate samples)
+            eido.validate_project(
+                new_project, "http://schema.databio.org/pep/2.1.0.yaml"
+            )
+        except Exception as e:
+            raise HTTPException(
+                status_code=400,
+                detail=f"",
```

Comment:
? no detail

Reply:
note sure about this from @khoroshevskyi